PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/webapp/webappconfig/slimui/WebAppSlimSettingPanel.java

```diff
@@ -268,23 +268,22 @@ public synchronized void fillWebApps(List<ResourceEx<WebApp>> webAppLists, final
     }
 
     @Override
-    public synchronized void fillDeploymentSlots(List<DeploymentSlot> slotList) {
+    public synchronized void fillDeploymentSlots(List<DeploymentSlot> slotList, final ResourceEx<WebApp> selectedWebApp) {
         cbxSlotName.removeAllItems();
         cbxSlotConfigurationSource.removeAllItems();
 
         cbxSlotConfigurationSource.addItem(Constants.DO_NOT_CLONE_SLOT_CONFIGURATION);
-        cbxSlotConfigurationSource.addItem(getSelectedWebApp().getResource().name());
+        cbxSlotConfigurationSource.addItem(selectedWebApp.getResource().name());
```

Comment:
null check for selectedWebApp?

Reply:
`fillDeploymentSlots` was invoked by `onLoadDeploymentSlots` in `WebAppDeployViewPresenterSlim`, where we did the null check, i'll mark the parameter as NotNull in later PR

Reply:
it is public, you will not member the convention,  and in future, you might call it will null selectedWebApp